.yarnclean

```diff
@@ -0,0 +1,2 @@
+# Aliased to clsx
+classnames
```

Comment:
This was only for testing purposes. It should stay in the bundle if some other part of the repo needs it.